zoning/data_processing/eval.py

```diff
@@ -196,6 +226,12 @@ async def evaluate_term(
         if num_correct_page_searched != 0
         else 0,
         "answer_accuracy": num_correct_answer / len(search_results_df),
+        "accuracy": (len(
+                search_results_df.filter(
+                    pl.all_horizontal(pl.col("correct_page_searched", "correct_answer"))
+                    > 0
+                )
+            ) / len(search_results_df))
```

Comment:
Makes sense to me. Can you add comments explaining the difference between each of these accuracy definitions to make sure things don't get too confusing?